trace/http_test.go

```diff
@@ -167,52 +167,17 @@ func TestSetTraceInHTTPRequest_EmptyTraceID(t *testing.T) {
 	assert.Equal(t, "", r.Header.Get(headerProbabilitySample))
 }
 
-func TestWithTraceFromHTTPRequest(t *testing.T) {
-	r, err := http.NewRequest("POST", "URL", nil)
-	assert.NoError(t, err)
-
-	r.Header.Add(headerTraceID, "00000000000000000000000000000019")
-	r.Header.Add(headerProbabilitySample, "0.5")
-
-	ctx := WithTraceFromHTTPRequest(context.Background(), r, map[string]string{
-		"k1": "v1",
-		"k2": "v2",
-	})
-
-	trace := GetTraceFromContext(ctx)
-
-	assert.Equal(t, "00000000000000000000000000000019", trace.ID.String())
-	assert.Equal(t, 0.5, *trace.ProbabilitySample)
+//
+// [DEPRECATED]
```

Comment:
Could you please add more information about the this? What is deprecated and to what it should be changed to, Whats the correct form?